pkg/util/validation/validation.go

```diff
@@ -226,6 +226,32 @@ func IsHTTPHeaderName(value string) bool {
 	return httpHeaderNameRegexp.MatchString(value)
 }
 
+func IsWhitelistedSysctlOpt(val string) bool {
+	validSysctlMap := map[string]bool{
```

Comment:
this should be global, not in the function. Otherwise, the list is recreated on every invocation.

I think `map[string]struct{}` is more idiomatic.